perun-wui-core/src/main/java/cz/metacentrum/perun/wui/client/utils/Utils.java

```diff
@@ -1389,4 +1420,29 @@ public static String translateKerberos(String name) {
 
 	}
 
+
+	/**
+	 * Returns user's id
+	 *
+	 * @return user's id or null when error
+	 */
+	public static Integer getUserId(PlaceManager placeManager) {
```

Comment:
If this shared method expect, that ID in URL params is always users ID, you should probably move it to some profile utils class. Because it won't generally work in other apps and javadoc doesn't mention that.